### PR TITLE
feat(privacy_policy) : add privacy policy page

### PR DESCRIPTION
**Pull request by JorisPotier**

### CODE REVIEW

File: privacy_policy.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+<p>Date de prise d'effet: September 24, 2018</p>
+
+
+<p>Code-Troopers ("nous", "notre", "nos") exploite le site web et l'application mobile Chapitô (ci-après désignés par le
+    terme "Service").</p>
+
+<p>Cette page vous explique nos politiques en matière de collecte, d'utilisation et de communication des données à
+    caractère personnel lorsque vous utilisez notre Service ainsi que les choix qui s'offrent à vous en ce qui concerne
+    ces données. <a href="https://privacypolicies.com/free-privacy-policy-generator/">Politique de Confidentialité via
```
Comment (age2pierre) [inline privacy_policy.html:27]: Did you left that link purposefully ? If so, ignore me

File: privacy_policy.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        <p>Firebase est un service d'analyse fourni par Google Inc.</p>
+        <p>Vous pouvez refuser certaines fonctionnalités de Firebase en utilisant les paramètres de votre dispositif
+            mobile, par exemple ses paramètres publicitaires, ou en suivant les instructions qui figurent dans la
+            politique de confidentialité de Google: <a href="https://policies.google.com/privacy?hl=en">https://policies.google.com/privacy?hl=en</a>
+        </p>
+        <p>Nous vous encourageons également à consulter la politique de Google relative à la protection de vos données:
+            <a href="https://support.google.com/analytics/answer/6004245">https://support.google.com/analytics/answer/6004245</a>.
+        </p>
+        <p>Pour plus de précisions sur le type de données recueillies par Firebase, merci de consulter la page web
+            Protection de la vie privée et conditions de Google <a href="https://policies.google.com/privacy?hl=en">https://policies.google.com/privacy?hl=en</a>
```
Comment (age2pierre) [inline privacy_policy.html:195]: Shouldn't we link to a more firebase specific page :
https://support.google.com/firebase/answer/6318039
or
https://firebase.google.com/support/privacy/ ?

File: privacy_policy.html (hunk 1 above)
Comment (JorisPotier) [reply privacy_policy.html:27]: yes

File: privacy_policy.html (hunk 2 above)
Comment (JorisPotier) [reply privacy_policy.html:195]: maybe

**Pull request merged**